srcDoc/release_notes.rst

```diff
@@ -79,6 +79,13 @@ which return the time in *microseconds*.
 This option can be enabled at compilation time by using the `--ns-resolution`.
 It is only implemented for Unix Systems.
 
+Added support for -threadPriorities command line parameter in QNX platforms.
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
Could you set the length of the ^^^^^^^^^^ line to the same number of characters of the tittle (this is silly, but it gives you a warning when compiling the documentation).

Reply:
Sorry I wasn't aware of this. Now it is fixed.